npm/ng-packs/apps/dev-app/src/app/home/home.component.ts

```diff
@@ -1,19 +1,25 @@
 import { AuthService } from '@abp/ng.core';
 import { Component } from '@angular/core';
 import { OAuthService } from 'angular-oauth2-oidc';
+import { timeInterval } from 'rxjs';
 
 @Component({
   selector: 'app-home',
   templateUrl: './home.component.html',
 })
 export class HomeComponent {
+  isLog: boolean = false;
   get hasLoggedIn(): boolean {
     return this.authService.isAuthenticated;
   }
 
   constructor(private authService: AuthService) {}
 
   login() {
-    this.authService.navigateToLogin();
+    this.isLog = true;
+    setTimeout(data => {
```

Comment:
Nice idea but we don't need a timeout.  Maybe you can catch navigation end or navigation error.